src/models/transaction.model.ts

```diff
@@ -0,0 +1,36 @@
+import { TransactionTypeEnum } from '../utils/enums';
+
+const mongoose = require('mongoose');
+const transactionSchema = mongoose.Schema(
+  {
+    amount: {
+      type: Number,
+      required: true,
+    },
+    from: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'User',
```

Comment:
> from and to should be ref with payment (from one payment card to another payment cart), cause payment should be between two card but not user.

user is alright, because if user does not choose to fill account bank manually instead of available ones, payment is null there